app/views/lightweight_activities/_enable_disable_runtime_fields.html.haml

```diff
@@ -30,9 +30,10 @@
     })
 
     $view_edit_glossary.on('click', function (e) {
+      var edit_url = "#{edit_glossary_path(-1)}"
```

Comment:
Add a comment about what the `-1` is for

Reply:
Fixed in https://github.com/concord-consortium/lara/pull/931